package_build_list_host_darwin.json

```diff
@@ -0,0 +1,65 @@
+        "OpenSSL-1.1.1b-rev2-mac": "package-system/OpenSSL/build_package_image.py",
+        "OpenSSL-1.1.1b-rev2-ios": "package-system/OpenSSL/build_package_image.py --platform ios",
+        "PhysX-4.1.2.29882248-rev3-mac": "package-system/PhysX/build_package_image.py --platform mac",
+        "PhysX-4.1.2.29882248-rev3-ios": "package-system/PhysX/build_package_image.py --platform ios",
+        "SPIRVCross-2021.04.29-rev1-mac": "Scripts/extras/pull_and_build_from_git.py ../../package-system/SPIRVCross --platform-name Mac --package-root ../../package-system --clean",
+        "DirectXShaderCompilerDxc-1.6.2104-o3de-rev2-mac": "Scripts/extras/pull_and_build_from_git.py ../../package-system/DirectXShaderCompiler --platform-name Mac --package-root ../../package-system  --clean",
+        "tiff-4.2.0.10-mac" : "package-system/tiff/build_package_image.py --platform mac",
+        "tiff-4.2.0.10-ios" : "package-system/tiff/build_package_image.py --platform ios",
+        "python-3.7.10-rev1-darwin" : "python/build_package_image.py",
+        "mcpp-2.7.2_az.1-rev1-mac": "package-system/mcpp/get_and_build_mcpp.py mcpp-2.7.2_az.1-rev1",
+        "mikkelsen-1.0.0.4-mac": "mikkelsen/build_package_image.py --platform mac",
+        "mikkelsen-1.0.0.4-ios": "mikkelsen/build_package_image.py --platform ios"  
+    },
+    "build_from_folder" : { 
+        "AWSNativeSDK-1.7.167-rev3-mac" : "package-system/AWSNativeSDK-mac",
+        "AWSNativeSDK-1.7.167-rev3-ios" : "package-system/AWSNativeSDK-ios",
+        "Lua-5.3.5-rev6-mac": "package-system/Lua-mac",
+        "Lua-5.3.5-rev5-ios": "package-system/Lua-ios",
+        "AwsIotDeviceSdkCpp-1.12.2-rev1-mac": "package-system/AwsIotDeviceSdkCpp-mac",
+        "etc2comp-9cd0f9cae0-rev1-mac" : "package-system/etc2comp-mac",
+        "freetype-2.10.4.9-mac" : "package-system/FreeType2-mac",
+        "freetype-2.10.4.9-ios" : "package-system/FreeType2-ios",
+        "googlebenchmark-1.5.0-rev2-ios": "package-system/googlebenchmark-ios",
+        "googlebenchmark-1.5.0-rev2-mac": "package-system/googlebenchmark-mac",
+        "googletest-1.8.1-rev4-ios": "package-system/googletest-ios",
+        "googletest-1.8.1-rev4-mac": "package-system/googletest-mac",
+        "libsamplerate-0.2.1-rev2-mac": "package-system/libsamplerate-mac",
+        "libsamplerate-0.2.1-rev2-ios": "package-system/libsamplerate-ios",
+        "OpenSSL-1.1.1b-rev2-mac": "package-system/OpenSSL-mac",
+        "OpenSSL-1.1.1b-rev2-ios": "package-system/OpenSSL-ios",
+        "tiff-4.2.0.10-mac" : "package-system/tiff-mac",
+        "tiff-4.2.0.10-ios" : "package-system/tiff-ios",
+        "python-3.7.10-rev1-darwin" : "python/darwin_x64/package",
+        "asn1-0.9.27-rev2-ios" : "package-system/asn1-ios",
+        "ASTCEncoder-2017_11_14-rev2-multiplatform" : "package-system/ASTCEncoder-multiplatform",
+        "PhysX-4.1.2.29882248-rev3-mac": "package-system/PhysX-mac",
+        "PhysX-4.1.2.29882248-rev3-ios": "package-system/PhysX-ios",
```

Comment:
This is not the latest from perforce. Please sync the latest. It's missing NvCloth, poly2tri and v-hacd libraries as build from source for darwin, linux and windows.

Reply:
I intend to sync with latest after this initial commit.   I need to get a working system in first (this snapshot is from like, a week  or two ago and I know all the packages actually work with the scripts).

Once this is in it is my intention to then get latest from p4 and do another PR to get it up to date.